components/interactive_dialog/dialog_element.jsx

```diff
@@ -166,6 +168,17 @@ export default class DialogElement extends React.PureComponent {
                     onChange={onChange}
                 />
             );
+        } else if (type === 'radio') {
+            return (
+                <RadioSetting
+                    id={name}
+                    label={displayNameContent}
+                    helpText={helpTextContent}
+                    values={options}
+                    value={value || options[0].value}
```

Comment:
This will fail if `options[0]` is undefined. Better if check `options` and `options[0]` are valid values.